react/Viewer/helpers.js

```diff
@@ -203,3 +203,13 @@ export const makeWebLink = ({ client, slug, path }) => {
     return null
   }
 }
+
+/**
+ * Remove the file name at the end of a path
+ * @param {string} path
+ * @returns new path
```

Comment:
```suggestion
 * @returns {string} new path
```

nit(you can add //@ts-check at the top of the file ;))